test/site-search.spec.js

```diff
@@ -92,7 +92,7 @@ describe('Site Search', () => {
 		});
 
 		it('should try to set the location to the catalog URL with a search term', () => {
-			const expected = 'https://ils-test.bcpl.lib.md.us/polaris/search/searchresults.aspx?term=harry%20potter';
+			const expected = 'https://ils-test.bcpl.lib.md.us/polaris/view.aspx?keyword=harry%20potter';
```

Comment:
This appears to be the only functional change in this PR (not including the removal of redundant conditional checks).  Everything else is very obviously cleanup.